jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  testEnvironment: "node",
+  testMatch: ["<rootDir>/tests/**/*.(spec|test).js"],
+  coveragePathIgnorePatterns: ["/node_modules/"],
```

Comment:
`jest.config` 파일을 따로 빼내는 것 보다는 `package.json`에 명시해주는 것은 어떨까요?
```suggestion
    "jest": {
    "testMatch": [
      "<rootDir>/tests/*.test.js"
    ]
  }
```

Reply:
util 함수, 엔드포인트 기준으로 백엔드 유닛테스트를 작성하라고 하셔서 util폴더를 만들었습니다. 소켓 관련 테스트와 그외의 게임로직 관련 테스트을 포함할 수 있을 것 같습니다. 더 적절한 폴더 구조가 있다면 수정하겠습니다.

Reply:
jest config 설정을 `package.json`에 명시하는 걸로 수정하겠습니다. jest 셋업을 하면서 jest config 설정을 모아놓은 파일이 보통은 있어서 파일을 만들었습니다.

Reply:
저도 package.json에서 명시해주는 것이 좋을 것 같아요 !